awx/main/credential_plugins/hashivault.py

```diff
@@ -85,8 +89,21 @@
     'type': 'string',
     'help_text': _('Valid principals (either usernames or hostnames) that the certificate should be signed for.'),
 }]
-hashi_ssh_inputs['required'].extend(['public_key', 'role'])
+hashi_ssh_inputs['required'].extend(['public_key', 'secret_path', 'role'])
 
+hashi_token_inputs = copy.deepcopy(base_inputs)
+hashi_token_inputs['metadata'] = [{
+    'id': 'policies',
+    'label': _('Policies'),
+    'type': 'string',
+    'help_text': _('A list of policies for the token. This must be a subset of the policies belonging to the token making the request, unless root. If not specified, defaults to all the policies of the calling token')
```

Comment:
It might be good to clarify that this should be specified as a comma-delimited string.

Reply:
I will add some clarification and remove the useless required line.